template/README.rst

```diff
@@ -0,0 +1,40 @@
+<demo_name> Demo
+=============
+Short blurb on what this demo does.
+
+Usage
+-----
+A simple command on how to run the demo. For example,
+::
+  python <demo_name>.py
+
+Code Overview
+-------------
+Give a general overview of how the code works.
+
+When possible, use bullet points so that:
```

Comment:
Suggest reducing to something like "Prefer bite-sized descriptions in bullet points"